routes/users.js

```diff
@@ -1,29 +1,90 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
+const csrf = require('csurf');
+const passport = require('passport');
 
-/* GET users listing. */
-router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+let Order = require('../models/order');
+let Cart = require('../models/cart');
+
+let csrfProtection = csrf();
```

Comment:
Best Practice: ```csrfProtection``` has a poor semantic value. Consider alternative naming.

Reply:
create an issue, didn't address it yet https://github.com/piazzaana/chicken-fries/issues/63

Reply:
Consider, in the future, grouping like issues together into one to reduce the amount of 'paperwork' (creating issues and checking them off as closed).